www/_template/reference/configuration.md

```diff
@@ -5,110 +5,283 @@ description: The Snowpack configuration API reference.
+
+Previously config.cwd
+
+## config.install
+
+**Type**: `string[]`
+
+Known dependencies to install with Snowpack.
+
+> Used for installing packages any dependencies that couldn't be detected by our automatic import scanner (ex: package CSS files).
+
+## config.extends
+
+**Type**: `string`
+
+Inherit from a separate "base" config.
+
+> Can be a relative file path, an npm package, or a file within an npm package. Your configuration will be merged on top of the extended base config.
+
+## config.exclude
+
+**Type**: `string[]`
+
+**Default**: `['**/node_modules/**/*', '**/web_modules/**/*', '**/.types/**/*']`
+
+Exclude any files from the Snowpack pipeline.
+
+Supports glob pattern matching.
+
+
+## config.mount
+
+```
+mount: {
+  [path: string]: string | {url: string, resolve: boolean, static: boolean}
+}
+```
+
+Mount local directories to custom URLs in your built application.
+
+```js
+// snowpack.config.js
+// Example: Basic "mount" usage
+{
+  "mount": {
+    "src": "/dist",
+    "public": "/"
+  }
+}
+```
+
+You can further customize this the build behavior for any mounted directory by using the expanded object notation:
+
+ <!-- snowpack/src/config.ts -->
+
+```js
+// snowpack.config.js
+// Example: expanded object notation "mount" usage
+{
+  "mount": {
+    // Same behavior as the "src" example above:
+    "src": {url: "/dist"},
+    // Mount "public" to the root URL path ("/*") and serve files with zero transformations:
+    "public": {url: "/", static: true, resolve: false}
+  }
+}
+```
+
+- `mount.url` | `string` | _required_ : The URL to mount to, matching the string in the simple form above.
+- `mount.static` | `boolean` | _optional_ | **Default**: `false` : If true, don't build files in this directory. Copy and serve them directly from disk to the browser.
+- `mount.resolve` | `boolean` | _optional_ | **Default**: `true`: If false, don't resolve JS & CSS imports in your JS, CSS, and HTML files. Instead send every import to the browser, as written.
+-
+
+## config.alias
+
+**Type**: `object` (package: package or path)
+
+Configure import aliases for directories and packages.
+
+> Note: In an older version of Snowpack, all mounted directories were also available as aliases by **Default**. As of Snowpack 2.7, this is no longer the case and no aliases are defined by **Default**.
+
+```js
+// snowpack.config.js
+// Example: alias types
+{
+  alias: {
+    // Type 1: Package Import Alias
+    "lodash": "lodash-es",
+    "react": "preact/compat",
+    // Type 2: Local Directory Import Alias (relative to cwd)
+    "components": "./src/components"
+    "@app": "./src"
+  }
+}
+```
+
+## config.plugins
+
+**Type**: `array` containing pluginName `string` or an array [`pluginName`, {`pluginOptions`}
+
+Enable Snowpack plugins and their options
+
+```js
+// snowpack-config.js
+// Example: enable plugins both simple and expanded
+{
+  plugins: ['plugin-1', ['plugin-2', { 'plugin-option': false }]];
```

Comment:
can you explain simple and expanded a bit more here, similar to how we do in the alias section?

```
{
  plugins: [
    // Simple format: no options needed
    'plugin-1', 
    // Expanded format: allows you to pass options to the plugin
    ['plugin-2', { 'plugin-option': false }]
  ]
```